extension/custom.css

```diff
@@ -27,3 +28,8 @@ tt,
 .wiki-wrapper .wiki-history .commit-meta code {
 	font-family: 'Fira Code', 'Ubuntu Mono', 'Droid Sans Mono', 'Operator Mono', 'Liberation Mono', 'Source Code Pro', Menlo, Consolas, Courier, monospace !important;
 }
+
+.CodeMirror-lines pre.CodeMirror-line {
+	-webkit-font-variant-ligatures: normal;
```

Comment:
I don't think the `-webkit-` prefix is needed at all. This is a Chrome extension and it ought to work on Chrome without the prefix. The "Can I use" website doesn't list any browser supporting the prefixed property - http://caniuse.com/#feat=kerning-pairs-ligatures